src/core/src/basics/sample.cpp

```diff
@@ -196,6 +196,7 @@ bool Sample::apply_loops( const Loops& lo )
 		ERRORLOG( QString( "count %1 < 0 is not allowed" ).arg( lo.count ) );
 		return false;
 	}
+	//if( lo == __loops ) return true;
```

Comment:
No...well yes...but no...

The line was commented out anyway - but you removed it all together in
https://github.com/blablack/hydrogen/commit/5b20e0543aaa31d9814250796c339cd76c2d0dea


I looks like it didn't go through during a merge...

On Sat, Mar 26, 2016 at 11:29 AM Sebastian Moors <notifications@github.com>
wrote:

> In src/core/src/basics/sample.cpp
> <https://github.com/hydrogen-music/hydrogen/pull/291#discussion_r57512542>
> :
>
> > @@ -196,6 +196,7 @@ bool Sample::apply_loops( const Loops& lo )
> >  		ERRORLOG( QString( "count %1 < 0 is not allowed" ).arg( lo.count ) );
> >  		return false;
> >  	}
> > +	//if( lo == __loops ) return true;
>
> Is the comment intentional?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/hydrogen-music/hydrogen/pull/291/files/d9cf06ecea6c676553607db7cc76f72e376935a6#r57512542>
>